src/FsAutoComplete.Core/CodeGeneration.fs

```diff
@@ -515,13 +515,11 @@ module CodeGenerationUtils =
       // is an abstract member
       m.IsDispatchSlot
       // this member doesn't implement anything
-      && m.ImplementedAbstractSignatures.Count = 0
+      && (try m.ImplementedAbstractSignatures <> null &&  m.ImplementedAbstractSignatures.Count = 0 with _ -> true) // exceptions here trying to acces the member means we're safe
```

Comment:
wrapping this potentially-throwing access means that members like `Stream.Dispose(): unit`, which are Sealed, get though. But I don't see a way around this without FCS changes.